diagnostic.js

```diff
@@ -30,25 +30,68 @@ mongoose.connect('mongodb://localhost/mongoose-crud');
 
 /// ADD YOUR CODE BELOW
 
-const create = (name, description, startYear, endYear) => {};
+'use strict';
+
+const models = require('app/models');
+const Movement = models.movement;
+
+const create = (name, description, startYear, endYear) => {
+  Movement.create(name, description, startYear, endYear)
```

Comment:
This is close, but not quite right - `.create` is expecting an object. 
i.e.
```
{
  name: name,
  description: description, 
  startYear: startYear,
  endYear: endYear
}
```
Using the new ES2015 syntax, you could turn that list into an object by wrapping it in curly braces:
`{name, description, startYear, endYear}`